src/recruiting/router.js

```diff
@@ -1,14 +1,437 @@
-const express = require('express');
+const express = require("express");
 const router = express.Router();
 
+var Student = require("./models/potential-student").PotentialStudent;
+var Teacher = require("./models/teacher").Teacher;
+var Meeting = require("./models/meeting").Meeting;
 /**
  * @swagger
  * /students:
  *    get:
  *      description: Return list of students
+ *      produces:
+ *        - application/json
+ *      responses:
+ *        201:
+ *          description: Return created students
+ *        400:
+ *          description: Something unexpected went wrong
  */
+
 router.get("/students", (req, res) => {
-    return res.send("Hello");
+  Student.find({})
+    .then(students => {
```

Comment:
Dit geldt voor meerdere routes  maar wat als students null is? Alle andere services hebben null checks vgm